bundles.rst

```diff
@@ -6,7 +6,7 @@ The Bundle System
 .. caution::
 
     In Symfony versions prior to 4.0, it was recommended to organize your own
-    application code using bundles. This is no longer recommended and bundles
+    application code using bundles. This is `no longer recommended <https://symfony.com/doc/current/best_practices.html#don-t-create-any-bundle-to-organize-your-application-logic>`_ and bundles
```

Comment:
```suggestion
    application code using bundles. This is :ref:`no longer recommended <best-practice-no-application-bundles>`_ and bundles
```

and in `best_practices.rst` add the following before the headline we want to link to:

```rst
.. _best-practice-no-application-bundles::
```

Reply:
I hope that I finally fixed it right.

Reply:
looks very good, thank you for the update 👍